src/AccessibilityInsights.SharedUx/Controls/TestTabs/AutomatedChecksControl.xaml.cs

```diff
@@ -873,10 +874,6 @@ private void GroupItem_PreviewKeyDown(object sender, KeyEventArgs e)
                 {
                     exp.IsExpanded = true;
                 }
-                else
-                {
-                    urlLink?.Focus();
-                }
                 e.Handled = true;
             }
             else if (e.Key == Key.Left)
```

Comment:
we might be able to remove HyperLink block on line 884

Reply:
Yup. Removing.